sunpy/instr/iris/iris.py

```diff
@@ -54,6 +54,6 @@ def SJI_to_cube(filename, start=0, stop=None, hdu=0):
     # Set the date/time
 
     for i, m in enumerate(iris_cube):
-        m.meta['DATE-OBS'] = splits[i].center.isoformat()
+        m.meta['DATE-OBS'] = splits[i].center.iso
```

Comment:
I think it should be `.isot` here?

From https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat, 
> The optional argument sep (default 'T') is a one-character separator, placed between the date and time portions of the result.